script/fix_claim_reference.rb

```diff
@@ -0,0 +1,16 @@
+require File.expand_path('../../config/environment',  __FILE__)
+
+
+Card::Auth.as_bot do
+
+  claim_cards = Card.search :type_id=>Card::ClaimID,:not=>{:right_plus=>[Card[:source].name,{:refer_to=>{:type_id=>Card::SourceID} }]}
+  claim_cards.each do |card|
+    claim_source = card.fetch :trait=>:source
+    if claim_source
+      puts claim_source.name
```

Comment:
actually, I guess it matters just a tiny bit, in that we might need to reuse this in the future if the reference problem recurs.  If you do a little clean up, I think it's ready to merge and run.